source/frontend/src/features/mapSideBar/disposition/models/DispositionSaleContactModel.ts

```diff
@@ -0,0 +1,74 @@
+import { isNumber } from 'lodash';
+
+import {
+  fromApiOrganization,
+  fromApiPerson,
+  IContactSearchResult,
+} from '@/interfaces/IContactSearchResult';
+import { Api_DispositionSaleContact } from '@/models/api/DispositionFile';
+
+export interface WithSalePurchasers {
+  dispositionPurchasers: DispositionSaleContactModel[];
+}
+
+export class DispositionSaleContactModel {
+  contact: IContactSearchResult | null = null;
+  primaryContactId: string = '';
+
+  constructor(
+    readonly id: number | null = null,
+    readonly dispositionSaleId: number | null = null,
+    readonly rowVersion: number | null = null,
+    contact: IContactSearchResult | null = null,
+  ) {
+    this.id = id;
+    this.dispositionSaleId = dispositionSaleId;
+    this.contact = contact;
+    this.rowVersion = rowVersion;
+  }
+
+  toApi(): Api_DispositionSaleContact | null {
```

Comment:
please don't use the old API models anymore. instead the generated ones to avoid type missmatches

Reply:
Updated to new models